doc/source/reference/apis/metadata.md

```diff
@@ -10,6 +10,10 @@ This means that we are currently exploring the best possible interface and funct
 As a warning word this means that the API or the way you define metadata may be subject to change before
 this feature graduates. If you have any comments or suggestion please open the issue on our GitHub project.
 
+Incubating update 1:
```

Comment:
What would be the best way to give these updates?

Maybe we should just move these to the release notes? We can treat metadata as its own "area", kind of like how Kubernetes handles it, and just add these on the `1.2` changelog under a `Metadata` section.

Reply:
Hmm... Not sure if actually that note is needed... that is not breaking much of things. 

Just cannot remember if graph-metadata was part of 1.1? If no, it's probably adding section about it there indeed.

Reply:
It would be good for the release changelog to also include new things. Not necessarily just breaking changes.